ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -2190,6 +2245,7 @@ def main():
             vm_set_name=dict(required=False, type='str'),
             topo=dict(required=False, type='dict'),
             vm_names=dict(required=True, type='list'),
+            vm_name=dict(required=False, type='list'),
```

Comment:
This new `vm_name` argument is confusing to me. Why it is a `list`? What's the difference between the `vm_names` argument. What is its purpose?

Reply:
If `vm_name` is only for a single value, why need to define it as a `list`? Why it needs to be consistent with the structure of `vm_names`? Consistency of variable name and type is important for readability. I have 2 concerns here:
1. I don't understand why `vm_name` must be a list.
2. The variable name `vm_name` and its type `list` do not really match from readability's point of view.